src/index.html

```diff
@@ -1,14 +1,27 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="flex-container">
+          <a href="" class="category is-active">APPLE</a>
+          <a href="" class="category">SAMSUNG</a>
+          <a href="" class="category">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
```

Comment:
Не залишай пусті href
```suggestion
          <a href="#" class="category">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
```